libraries/networking/src/AssetClient.cpp

```diff
@@ -70,11 +70,12 @@ AssetUpload* AssetClient::createUpload(const QString& filename) {
     auto nodeList = DependencyManager::get<NodeList>();
     SharedNodePointer assetServer = nodeList->soloNodeOfType(NodeType::AssetServer);
     
-    if (assetServer) {
-        return new AssetUpload(this, filename);
+    if (!assetServer) {
+        qDebug() << "No Asset Server";
```

Comment:
make this more descriptive in case someone catches it in the log - "Could not upload $filename since you are not currently connected to an asset-server."